pkg/services/user/model.go

```diff
@@ -138,6 +147,8 @@ type UserProfileDTO struct {
 	Login                          string          `json:"login"`
 	Theme                          string          `json:"theme"`
 	OrgID                          int64           `json:"orgId,omitempty"`
+	Level                          int64           `json:"level,omitempty"`
```

Comment:
@drew08t i think reason why level wasn't being sent when 0 is due to the `omitempty` tag here 🤔 just for future reference :)